internal/configuration/template.go

```diff
@@ -0,0 +1,37 @@
+package configuration
+
+import (
+	_ "embed"
+	"fmt"
+	"io/ioutil"
+	"os"
+)
+
+//go:embed config.template.yml
+var template []byte
+
+// EnsureConfigurationExists is an auxilery function to the main Configuration tools that ensures the Configuration
+// template is created if it doesn't already exist.
+func EnsureConfigurationExists(paths []string) (created bool, err error) {
```

Comment:
```suggestion
func EnsureConfigurationExists(path string) (created bool, err error) {
```
I'd rather move the below check in the callers function in order for the signature of this function to be more intuitive (check the existence of one configuration instead of check the existence of the first configuration in the list of configurations).

Reply:
Committed.